payment/src/main/java/org/killbill/billing/payment/core/sm/control/ControlPluginRunner.java

```diff
@@ -117,7 +119,7 @@ public PriorPaymentControlResult executePluginPriorCalls(final Account account,
                 inputPluginProperties = prevResult.getAdjustedPluginProperties();
             }
             if (prevResult.isAborted()) {
-                break;
+                throw new PaymentApiException(ErrorCode.PAYMENT_PLUGIN_API_ABORTED, pluginName);
             }
```

Comment:
Yeah, for getting the reason, we'd need to change the ControlPlugin API. We could do it, but it seems like a change with a broader scope...

For the error message, should we change here:
https://github.com/killbill/killbill-api/blob/master/src/main/java/org/killbill/billing/ErrorCode.java#L258

Or just use a custom message while constructing the exception instance?